_posts/2023-06-11-vanilla-os-orchid-devlog.md

```diff
@@ -0,0 +1,195 @@
+
+![System composition of Vanilla OS](/assets/uploads/images.webp)
+
+This division allows us to open the doors to every type of flavor, ranging from servers to desktops. It would also allow you to change your base system from Debian to your distribution of choice.
+
+## Apx v2 and VSO v2
+
+Apx and VSO (Vanilla System Operator) have always been different utilities with nothing in common. It is changing in Vanilla OS Orchid, where the two tools collaborates to cover a wide range of use cases.
+
+### Apx, as a developer tool
+
+Apx v2 was designed primarily for developers and anyone who needs isolated and easily manageable environments, with integration with Vanilla OS.
+
+In Apx v1, it was possible to use multiple package managers by specifying the dedicated flag, such as `--dnf` for Fedora or `--zypper` for openSUSE, making it easy to interact with packages from other Linux distributions.
+
+Apx v2 still offers this possibility, but it does so through stacks that are configurable environments based on the user's needs.
+
+A stack is a set of instructions that can configure an environment based on any Linux distribution, adding packages and settings at the user's discretion. At any time, the user can create a new stack by defining a base Linux distribution, such as Arch Linux, and an optional list of packages. With this stack, the user can start one or more new subsystems based on that stack.
+
+Once a subsystem gets created, the user can use package manager commands (install, remove, update, etc.) or enter the subsystem, just like in Apx v1.
+
+Stacks are shareable and restoreable, so you can easily share them with your team. Apx v2 already has a set of preconfigured stacks to access all the major Linux distributions.
+
+Thanks to this new structure, the user can create stacks for each use case, for example, one with all the tools and libraries useful for development in various domains like 3D applications, music production, programming, etc.
+
+If something goes wrong, the subsystem can be restored by reinitializing it, resetting the environment to the state defined by the stack.
+
+### VSO, as a system package manager
+
+VSO v2 plays a crucial role in Vanilla OS Orchid. On top of managing update configurations, it will also handle the installation of Deb, Nix, and Apk packages through Waydroid.
+
+#### Vanilla OS Subsystem
+
+Since Apx v2 is a tool designed for developers and more advanced users, which will receive several other features for this target audience over time, we have decided to create a new subsystem specifically for the end user, who is typically less experienced. In VSO v2, there is a subsystem based on Vanilla OS Pico. This subsystem will be a SystemD container and is deeply integrated into the system, allowing the most simple installation, removal, and management of packages, both graphically and from the command line.
```

Comment:
```suggestion
Since Apx v2 is a tool designed for developers and more advanced users, which will receive several other features for this target audience over time, we have decided to create a new subsystem specifically for the end user, who is typically less technologically inclined. In VSO v2, there is a subsystem based on Vanilla OS Pico. This subsystem will be a SystemD container deeply integrated into the system, allowing straightforward installation, removal, and management of packages, both graphically and through the command line.
```